src/contact.js

```diff
@@ -0,0 +1,23 @@
+export default function () {
+  const pageContent = document.querySelector('.page-content');
+  pageContent.innerHTML = '';
+  const div = document.createElement('div');
+  div.className = 'columns is-centered';
+  const contactCard = `
+    <div class="column is-half">
+      <div class="content has-text-centered">
+        <p><strong>Find us on all social media and follow us!</strong></p>
+        <br>
+        <p>For party orders or use of the hall please send us an email</p>
+        <p>party@bangaloretiffincenter.com</p>
+        <br>
+        <p>Phone Number</p>
+        <p>+91-9112764532</p>
+        <br>
+        <p>Address</p>
+        <p>Number 10, Shri Kala Building, Lavelle Road, Bengaluru-560012</p>
+      </div>
+    </div>`;
```

Comment:
- [ ] Prioritize the use of `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).
- Replace all the rest